supplementary_style_guide/style_guidelines/code-commands.adoc

```diff
@@ -29,14 +29,30 @@ If you include a shell prompt to indicate that a user with root privileges must
 [[ip-addresses-and-mac-addresses]]
 == IP addresses and MAC addresses
 
-Use the IP and MAC address ranges that are reserved for documentation purposes to avoid the likelihood of conflicts and confusion:
+Use the IP and MAC address ranges that are reserved for documentation purposes to avoid the likelihood of conflicts and confusion.
 
-* Reserved IP address ranges: See the _IBM Style_ guide for the list of reserved IP address ranges to use in documentation.
+[[reserved-ip-addresses]]
+=== Reserved IP addresses
 
-* Reserved MAC addresses (defined in link:https://www.rfc-editor.org/rfc/rfc7042.html#section-2.1.2[RFC 7042]):
+Reserved IPv4 addresses for documentation are defined in link:https://www.rfc-editor.org/rfc/rfc5737.html[RFC 5737]:
 
-** For unicast: 00:00:5E:00:53:00 - 00:00:5E:00:53:FF
-** For multicast: 01:00:5E:90:10:00 - 01:00:5E:90:10:FF
+* 192.0.2.0/24 (or 192.0.2.*)
```

Comment:
From looking at the RFC, I do not directly see information pertaining to using blocks other than 192.0.2.0/24, 198.51.100.0/24, and 203.0.113.0/24. The asterisk might indicated that one can use the address on any port, which might be OK?

Reply:
@dfitzmau I admit that i am no expert on IP addresses. I took these values from what the IBM style guide listed here: https://www.ibm.com/docs/en/ibm-style?topic=elements-web-ip-email-addresses#using-addresses-in-examples

@mmuehlfeldRH I know that you submitted the section on mac addresses. Are you familiar enough with IP addresses too to give some feedback on what we should list for IP addresses? I know that you gave an actual range for mac addresses

Reply:
> The asterisk might indicated that one can use the address on any port, which might be OK?

@dfitzmau In the network world, a "port" identifies an endpoint that a service uses (for example, HTTPS uses, by default, port 443). Services can listen on a specific port on a specific IP address in a host (e.g. 192.0.2.1:443), but this has nothing to do with IP addresses.


> Are you familiar enough with IP addresses too to give some feedback on what we should list for IP addresses?

@bergerhoffer If you want to show a range instead of 192.0.2.* (which is not a typical notation), then we can show: 192.0.2.0 - 192.0.2.255

Some additional information:

A /24 subnet mask in an IPv4 network means that the last octet ("block") of the address can be anything between 0 and 255, but the 0 indicates in this case the network itself and 255 is the broadcast address.

IPv4 example: In a 192.0.2.0/24 network:

- You can assign addresses from the range 192.0.2.1 - 192.0.2.254 to your hosts.
- 192.0.2.0 stands for the network itself. You must not assign this address to any device.
- 192.0.2.254 is the broadcast address in this network. You must not assign this address to any device.

It works the same for all IPv4 /24 networks.